src/mudlet.cpp

```diff
@@ -2500,3 +2505,9 @@ void mudlet::requestProfilesToReloadMaps( QList<QString> affectedProfiles )
 {
     emit signal_profileMapReloadRequested( affectedProfiles );
 }
+
+
+// addition to help with dock widget closing.
+void TDockWidget::closeEvent( QCloseEvent * event ) {
+    mudlet::self()->closeWindow( mudlet::self()->getActiveHost(), windowTitle() );
```

Comment:
Ok, that should be simple enough.  
I need to learn a bit more about the Hosts and what the best ways are to fetch and interact with them.